packages/sonarwhal/docs/contributor-guide/how-to/test-rules.md

```diff
@@ -17,9 +17,9 @@ If not, you need to:
 1. Have the following template:
 
 ```ts
-import { RuleTest } from '../../../helpers/rule-test-type'; // eslint-disable-line no-unused-vars
-import * as ruleRunner from '../../../helpers/rule-runner';
-import { getRuleName } from '../../../../src/lib/utils/rule-helpers';
+import { RuleTest } from 'sonarwhal/dist/tests/helpers/rule-test-type'; // eslint-disable-line no-unused-vars
+import * as ruleRunner from 'sonarwhal/dist/tests/helpers/rule-runner';
+import { getRuleName } from 'sonarwhal/dist/src/lib/utils/rule-helpers';
 
 const ruleName = getRuleName(__dirname);
```

Comment:
It is not very common, but if you create a package with multi rules, `getRuleName` doesn't work as expected. I will open an issue for that, but maybe the documentation should say something about that.